### PR TITLE
Make loki as an optional

### PR DESCRIPTION
**Pull request by clyang82**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -100,6 +100,8 @@ const (
 	middlewareTimeout
 )
 
+var enableLogs = true
```
Comment (periklis) [inline main.go:103]: Consider removing this global variable and group it under `logsConfig`. You can further enable this flag in `logsConfig` if at least one `{read,write,tail}Endpoint}` is provided.

Comment (clyang82) [reply main.go]: Done

File: main.go [hunk 2]
```diff
@@ -542,26 +546,40 @@ func parseFlags() (config, error) {
...
+		if err != nil {
+			return cfg, fmt.Errorf("--logs.tail.endpoint is invalid, raw %s: %w", rawLogsTailEndpoint, err)
+		}
+
+		cfg.logs.tailEndpoint = logsTailEndpoint
 	}
 
-	cfg.logs.writeEndpoint = logsWriteEndpoint
+	if rawLogsWriteEndpoint != "" {
+
```
Comment (squat) [inline main.go:573]: nit: extra newline here

Comment (clyang82) [reply main.go]: done

**Comment by clyang82** [comment]
@squat Do I need to commit `modules.txt` file? I remembered it has some difference in the past. but right now, in my local environment, there is no difference after running `make generate --always-make`

**Comment by squat** [comment]
spoke offline, the commit needed to remove the modules.txt from the PR, which was generated using go 1.13, causing an issue.

**Comment by squat** [comment]
lgtm, this simply needs a rebase now :)))

**Pull request merged**